definitions/control-v1.yaml

```diff
@@ -4503,6 +4503,21 @@ TOfReTlUQzgpXRW5h3n2LVXbXQhPGcVitb88Cm2R8cxQwgB1VncM8yvmKhREo2tz
           default: false
           description: If `true`, publishing messages with a push payload in the extras field is permitted and can trigger the delivery of a native push notification to registered devices for the channel.
           example: false
+        batchingEnabled:
+          type: boolean
+          default: false
+          description: If `true`, channels within this namespace will start batching inbound messages instead of sending them out immediately to subscribers.
+          example: false
+        batchingPolicy:
+          type: string
+          default: simple
+          description: If `true`, groups multiple incoming messages into a single batch.
+          example: simple
+        batchingInterval:
+          type: integer
+          default: simple
+          description: If true, the batching interval, configurable from 20ms to 1s, temporarily holds and aggregates incoming messages. These messages are then combined and delivered as a single batch once the interval elapses or a size limit is reached (before the interval elapses).
```

Comment:
```suggestion
          description: If `true`, the batching interval, configurable from 20ms to 1s, temporarily holds and aggregates incoming messages. These messages are then combined and delivered as a single batch once the interval elapses or a size limit is reached (before the interval elapses).
```

Reply:
Should the `true` be added to be consistent with the other descriptions?